runtime/src/runtime_api.rs

```diff
@@ -459,16 +459,54 @@ mod tests {
             .fold(0, |acc, x| acc.checked_add(*x).unwrap());
     }
 
-    #[ignore]
     #[test]
     fn call_size() {
```

Comment:
`bounty::Call` is missing in the list
Perhaps we should iterate over `AllPalletsWithSystem` if possible or at least add a sanity `core::mem::size_of::<Call>();` check of a "global" `Call` size in case someone forgets to add some new pallet here once created?

Reply:
Hmm, for some reason I'm getting different results now:
```
referendum: 56
forum: 72
constitution: 24
joystream_utility: 32
project_token: 72
proposals_discussion: 56
bounty: 168
storage: 120
working_group: 160
membership: 200
proposals_engine: 40
proposals_codex: 296
proposals_discussion: 56
content: 304
council: 96
----
Runtime: 312
```

Tested on commit [`3a5329c93dbd49d4a2bb02b83bca593ade51f8fc`](https://github.com/Joystream/joystream/pull/4336/commits/3a5329c93dbd49d4a2bb02b83bca593ade51f8fc)